webosose/ares-cli
# Update required permissions

:Release Notes:
Update requiredPermissions in templates

:Detailed Notes:
Update CLI template app's requiredPermissions in appinfo.json
- Replace new group name of service in appinfo.json
- Re-create/package jasmine test app

:Testing Performed:
Pass unit test on ose and auto
Pass eslint
Verify with CLI commands on device
- web app & js_service
1. Generate web app & js_service. Check appinfo.json
2. Package and install Ipk and launch it.
3. Click the app's screen. The text will be changed "Hello, webOS!"
4. Check web app's journalctl log. Sholuld has UTC log.
#journalctl|grep UTC
[APP_NAME: example web app] GETTIME_SUCCESS UTC : 1612395609"

- qml app 
1. Generate qml app and check appinfo.json
2. Package and install Ipk and launch it.
2. Click the app's screen. The text will be show "UTC" time

:Issues Addressed:
[PLAT-134318] Update CLI template app's requiredPermissions in appinfo.json
